lib/pmdtester/runner.rb

```diff
@@ -35,6 +36,13 @@ def run
       end
     end
 
+    def clean
+      clean_target = ['target/reports']
+      clean_target.each do |target|
```

Comment:
Do we need a loop here? I'd prefer to keep it simple for now and only add the loop, if we need to clean more...